HACKING.md

```diff
@@ -51,6 +51,16 @@ the tests/helpers directory.
 
 ## Verify ABRT plays nice with the rest of OS
 
+TL;DR: you can easily run a subset of the integration tests in a VM via [tmt](https://tmt.readthedocs.io/en/stable/).
+```shell
+# run tests in a Rawhide VM
+$ tmt run -vv
+# run tests in a F34 VM
+$ tmt run -vv --all provision --how virtual --image fedora-34
+```
+Note: You will need `tmt` and `tmt-provision-virtual` packages installed on your laptop.
+Also make sure that you have [virtualization configured](https://docs.fedoraproject.org/en-US/ci/tmt/#_virtualization_tips) properly.
```

Comment:
Actually, since `tmt-1.4` this is no more necessary, we're using qemu user mode so under regular user no additional actions should be needed.